OpenRA.Mods.Common/Widgets/Logic/Ingame/SupportPowerTooltipLogic.cs

```diff
@@ -65,11 +77,25 @@ public SupportPowerTooltipLogic(Widget widget, TooltipContainerWidget tooltipCon
 				hotkeyLabel.Visible = hotkey.IsValid();
 
 				var timeWidth = timeFont.Measure(time).X;
+				var costWidth = costFont.Measure(costString).X;
 				var topWidth = nameFont.Measure(name).X + hotkeyWidth + timeWidth + timeOffset;
+
+				if (cost != 0)
+				{
+					topWidth += costWidth + costOffset;
+				}
+
 				var descSize = descFont.Measure(desc);
 				widget.Bounds.Width = 2 * nameLabel.Bounds.X + Math.Max(topWidth, descSize.X);
 				widget.Bounds.Height = baseHeight + descSize.Y;
 				timeLabel.Bounds.X = widget.Bounds.Width - nameLabel.Bounds.X - timeWidth;
+
+				if (cost != 0)
```

Comment:
Done, but at the cost of the Cost label's X value being used for both left and right margins.